src/app/services/coffeeShops.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
```

Comment:
I totally agree with having a service for interacting with the coffee shops endpoint. :+1: